src/native/libs/build-native.sh

```diff
@@ -60,15 +60,15 @@ source "$__RepoRootDir"/eng/native/build-commons.sh
 
 # Set cross build
 if [[ "$__TargetOS" == browser ]]; then
-    if [[ -z "$EMSDK_PATH" ]]; then
+    if [[ -z "$EMSDK" ]]; then
```

Comment:
It would be better to just set `EMSDK_PATH` to `EMSDK` somewhere above s.t. these lines do not need to be modified.

Reply:
Done, thanks